Octokit/Models/Response/ResourceRateLimit.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+
+using Octokit.Helpers;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class ResourceRateLimit
```

Comment:
I'd rather we just use `RateLimit` and not this class. The name of this class is too similar to `ResourceRateLimits`. The data we show in this response and in the headers are the same and not likely to change (as we try and avoid breaking changes).